docs/conf_sso_test_msscolab.rst

```diff
@@ -14,63 +14,72 @@ To set up a local identity provider with the mscolab server, you'll first need t
 
-Then you can generate files, simply by running,
+If everything is correctly set, you can generate keys and certificates simply by running
 
 `$ python mslib/mscolab/mscolab.py sso_conf --init_sso_crts`
 
+.. note::
+    This process generating keys and certificates for both Identity provider and mscolab server by default, If you need configure with different keys and certificates for the Identity provider, You should manually update the path of `SERVER_CERT` with the path of the generated .crt file for Identity provider, and `SERVER_KEY` with the path of the generated .key file for the Identity provider in the file `MSS/mslib/idp/idp_conf.py`.
 
 
-Enable USE_SAML2
-----------------
+3. Enable USE_SAML2
+-------------------
 
-To enable saml2 based login (identity provider-based login), set `USE_SAML2 = True` in the `mslib/mscolab/conf.py` file of the MSS Colab server.
+To enable SAML2-based login (identity provider-based login), 
 
-After setting the `USE_SAML2`, the next step is to add the `CONFIGURED_IDPS` dictionary. This dictionary should include keys for each enabled Identity Provider, represented by `idp_identity_name`, and their corresponding `idp_name`. Once this dictionary is set up, it should be used to update various functionalities of the mscolab server, such as the SAML2Client config .yml file, ensuring proper integration with the enabled IDPs.
+- To start the process update `USE_SAML2 = True` in your `mscolab_settings.py` file.
 
+.. note::
+    After enabling the `USE_SAML2` option, the subsequent step involves adding the `CONFIGURED_IDPS` dictionary for the MSS Colab Server. This dictionary must contain keys for each active Identity Provider, denoted by their `idp_identity_name`, along with their respective `idp_name`. Once this dictionary is configured, it should be utilized to update several aspects of the mscolab server, including the SAML2Client configuration in the .yml file. This ensures seamless integration with the enabled IDPs. By default, configuration has been set up for the localhost IDP, and any additional configurations required should be performed by the developer.
 
-Generate metadata files
------------------------
+4. Generate metadata files
+--------------------------
 
-This involves generating necessary metadata files for both the identity provider and the service provider. You can generate them by simply running the appropriate command.
+This involves generating necessary metadata files for both the identity provider and the service provider. You can generate them by simply running the below command.
 
-Before executing this, you should set `USE_SAML2=True` as described in the third step(USE_SAML2).
+.. note::
+    Before executing this, you should set `USE_SAML2=True` as described in the third step(Enable USE_SAML2).
 
 `$ python mslib/mscolab/mscolab.py sso_conf --init_sso_metadata`
 
```

Comment:
see above

Reply:
you could for example check if we are in a git repo, then it will work for devs. And when it is installed there is an entry point.